File: Código-fonte/Web/Common/Models/Pacote.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Common.Models
{
    public class Pacote
    {
        public virtual Guid PacoteId { get; set; }
        public virtual string TagRFID { get; set; }
        public virtual string Codigo { get; set; }
        public virtual Guid Remetente { get; set; }
        public virtual Guid DestinatarioId { get; set; }
        public virtual string Destinatario { get; set; }
        public virtual ICollection<Rota> Rotas { get; set; }
        public virtual DateTime DataPostagem { get; set; }
        public virtual Guid Destino { get; set; }
        public virtual bool Entregue { get; set; }
    }
}

File: Código-fonte/App/app/src/main/java/br/usp/pcs2018/rastreamentopacotesapp/Models/Pacote.java
package br.usp.pcs2018.rastreamentopacotesapp.Models;

import java.util.ArrayList;
import java.util.Date;
import java.util.List;

public class Pacote {

    private String PacoteId;
    private String Remetente;
    private String Destinatario;
    private Date DataPostagem;
    private List<Rota> Rotas;
    private boolean Entregue;
    private Endereco Destino;

    public Pacote(){
        this.Rotas = new ArrayList<>();
    }

    public String getPacoteId() {
        return PacoteId;
    }

    public void setPacoteId(String pacoteId) {
        this.PacoteId = pacoteId;
    }

    public String getRemetente() {
        return Remetente;
    }

    public void setRemetente(String remetente) {
        this.Remetente = remetente;
    }

    public String getDestinatario() {
        return Destinatario;
    }

    public void setDestinatario(String destinatario) {
        this.Destinatario = destinatario;
    }

    public Date getDataPostagem() {
        return DataPostagem;
    }

    public void setDataPostagem(Date dataPostagem) {
        this.DataPostagem = dataPostagem;
    }

    public List<Rota> getRotas() {
        return Rotas;
    }

    public void setRotas(List<Rota> rotas) {
        this.Rotas = rotas;
    }

    public boolean isEntregue() {
        return Entregue;
    }

    public void setEntregue(boolean entregue) {
        Entregue = entregue;
    }

    public Endereco getDestino() {
        return Destino;
    }

    public void setDestino(Endereco destino) {
        Destino = destino;
    }
}
